subtests/docker_cli/build/build.py

```diff
@@ -483,93 +459,57 @@ def make_builds(self, source):
                               postproc_cmd_csv=postproc_cmd_csv)]
 
     def initialize(self):
-        super(build_base, self).initialize()
-        self.initialize_utils()
+        super(BuildSubSubtest, self).initialize()
         self.sub_stuff['builds'] = []
         # Side-effect: docker pulls the base-image
         self.sub_stuff['builds'] += self.make_builds(self.config)
-        pec = self.parent_subtest.stuff['existing_containers']
-        self.sub_stuff['existing_containers'] = pec
-        # Count after pulling base-image
-        pei = self.parent_subtest.stuff['existing_images']
-        self.sub_stuff['existing_images'] = pei
 
     def run_once(self):
-        super(build_base, self).run_once()
+        super(BuildSubSubtest, self).run_once()
         for build_def in self.sub_stuff['builds']:
             build_def.dockercmd.execute()
 
     def postprocess(self):
         try:
-            super(build_base, self).postprocess()
+            super(BuildSubSubtest, self).postprocess()
         except:
             for build_def in self.sub_stuff['builds']:
                 build_def.dockercmd.verbose = True
                 self.logdebug(str(build_def.dockercmd))
             raise
 
-    def cleanup(self):
-        super(build_base, self).cleanup()
-        # Some of this could have been modified, recover from source
-        self.parent_subtest.reset_build_context()
-        # Auto-converts "yes/no" to a boolean
-        if self.config['remove_after_test']:
-            # Remove all previously non-existing containers
-            for cid in self.sub_stuff['dc'].list_container_ids():
-                if cid in self.sub_stuff['existing_containers']:
-                    continue    # don't remove previously existing ones
-                self.logdebug("Removing container %s", cid)
-                dcmd = DockerCmd(self, 'rm', ['--force', '--volumes', cid])
-                dcmd.execute()
-            dimg = self.sub_stuff['di']
-            base_repo_fqin = DockerImage.full_name_from_defaults(self.config)
-            # Remove all previously non-existing images
-            for img in dimg.list_imgs():
-                if img in self.sub_stuff['existing_images']:
-                    continue
-                if img.full_name is None or img.full_name is '':
-                    thing = img.long_id
-                else:
-                    thing = img.full_name
-                    # never ever remove base_repo_fqin under any circumstance
-                    if thing == base_repo_fqin:
-                        continue
-                self.logdebug("Removing image %s", img)
-                dcmd = DockerCmd(self, 'rmi', ['--force', thing])
-                dcmd.execute()
-
-
-class local_path(build_base):
+
+class local_path(BuildSubSubtest):
```

Comment:
May I assume you're aware of the AttributeError? (I assume, but have not confirmed, that it's traceable to here)